### PR TITLE
S3 | Really Login a user (#43)

### PR DESCRIPTION
**Pull request by CalebSarikey**
Updated users-api/routes/login.js to really login a user (#43)

### CODE REVIEW

File: users-api/routes/login.js [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-    res.status(403).json
-    ({
-        "message": "Invalid Credentials"
+router.post('/', (req,res) => {
+  getDb().then(db => {
+    db.collection('users').findOne(
+      {username: req.body.username}, (err, user) => {
+        if(err) {
+          console.log('ERROR.')
+          res.json(err)
```
Comment (raghunat) [inline users-api/routes/login.js]: it should be more like `return res.status(500).json(err)` to have a bad status code, and prevent the rest of the function from running

Comment (CalebSarikey) [reply users-api/routes/login.js]: fixed.

**Comment by CalebSarikey** [comment]
fixed

File: users-api/routes/login.js [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+        if(err) {
+          console.log('ERROR.')
+          return res.status(500).json(err)
+        }
+        if (user && user.password === req.body.password){
+          console.log('Login successful.')
+          res.json({status: "Login successful."});
+        } else {
+          console.log("Invalid credentials.");
+          res.json({status: "Invalid credentials."});
```
Comment (raghunat) [inline users-api/routes/login.js]: You removed the status code, we should have it 403 when the creds are invalid

**Pull request merged**